examples/mobile/UIComponents/components/content/index.html

```diff
@@ -23,6 +23,46 @@ <h1>
 						Contents area
 					</a>
 				</li>
+				<li class="ui-li-anchor">
+					<a href="floatingactions/index.html">
+						Floating Actions
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="textinput.html">
+						Text Field
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="textenveloper/index.html">
+						Text Enveloper
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="searchinput/index.html">
+						Search Input
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="searchbar/index.html">
+						Search Bar
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="progress/index.html">
+						Progress &amp; Activity
```

Comment:
not need to &amp;
I think that you input only Progress.

Reply:
Done